Microsoft.DotNet.Try.Protocol.ClientApi/Project.cs

```diff
@@ -12,12 +12,16 @@ namespace Microsoft.DotNet.Try.Protocol.ClientApi
     [JsonConverter(typeof(ProjectJsonConverter))]
     public class Project : FeatureContainer
     {
+        private const string DefaultLanguage = "csharp";
```

Comment:
Is there an approach where we don't have the concept of a default language?

Reply:
+1

Reply:
is for backward compatibility